### ISSUE TITLE
generated html file not causing reload

### ISSUE THREAD

**Issue by omerts** [geelen/jspm-server#16]
Hello,

I have a package which generates an html file, after running unit tests. The jspm-server recognizes the change, but doesn't reload the page.

``` javascript
Change detected: karma\Chrome_43.0.2357_(Windows_7_0.0.0)\index.html
Change detected: karma\Chrome_43.0.2357_(Windows_7_0.0.0)\index.html
```

**Comment by geelen** [comment]
Hmm, that output is in your terminal, not the browser console, yes? What does your browser console output?

**Comment by omerts** [comment]
Yes the output is from the terminal. In the browser I only see the following:
![image](https://cloud.githubusercontent.com/assets/4996164/8329131/909745f8-1a7f-11e5-8ce5-c3bdae6e114e.png)

For what it's worth, live-server is reloading the page correctly. Also if I go to the site's root, the page does reload, when karma's index.html is changed.

**Comment by omerts** [comment]
I just notices, that any file, that is not the main index file, is not reloading on file change. Maybe I am doing something wrong?

**Comment by geelen** [comment]
I've just realised you must be on Windows. I haven't even tested there, I bet I'm doing something silly with `/` vs `\` :)
